### PR TITLE
sidebar scroll bar

### PR DESCRIPTION
**Pull request by Babacar02**
I fixed the issue where the sidebar would move when scrolling on the website, this may fix the mobile side or...
I also added the scrollbar which activates when enough routes are entered. Changed the color of the sidebar. 
Future: still want to add the clear all button and style to dynamically created routes / maybe a sidebar rework.

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -65,18 +71,41 @@ div#title-bar > h1 > .city {
...
 .sidebar {
-  height: 100%;
+  height: 90%;
   width: 0px;
   position: fixed;
   z-index: 1;
   top: 10vh;
-  left: 0;
-  background-color: rgb(226, 226, 226);
+  left:0;
```
Comment (riley-thomps) [inline src/styles.css:79]:
```suggestion
  left: 0;
```

File: src/styles.css [hunk 2]
```diff
@@ -65,18 +71,41 @@ div#title-bar > h1 > .city {
...
   padding-top: 15px;
   transition: 0.5s;
+  overflow-y: scroll;
+  scrollbar-color:  black; /* only works for firefox */
+  scrollbar-width: thin;
+}
+
+/* WebKit-based browsers (Chrome, Safari, Edge) */
+.sidebar::-webkit-scrollbar {
+  width:8px;
```
Comment (riley-thomps) [inline src/styles.css:91]:
```suggestion
  width: 8px; 
```

File: src/styles.css [hunk 3]
```diff
@@ -65,18 +71,41 @@ div#title-bar > h1 > .city {
...
 }
 
+.sidebar::-webkit-scrollbar-track {
+  background-color: transparent;
+}
+
+/** css sidebar color options
+soft mustard: #f0c75e
+off-white: #f9f9f9
+*/
```
Comment (riley-thomps) [inline src/styles.css:106]: I like the color you changed it to, the kinda egg-shell white you added looks very clean, better than the grey or mustard you sent in discord. So I think you can remove this comment.

File: src/styles.css [hunk 4]
```diff
@@ -103,6 +132,10 @@ div#title-bar > h1 > .city {
   color: #a71010;
 } */
 
+/* * sideebar scrollbar */
```
Comment (riley-thomps) [inline src/styles.css:135]:
```suggestion
/* * sidebar scrollbar */ 
```

**Comment by riley-thomps** [comment]
@Babacar02 I noticed an issue from testing your branch is that the `122 University Avenue Circulator` in the navbar is getting cut off by the scroll bar. You could go about this a couple ways:
- you could expand the navbar by a couple more pixels
- you could make it so the navbar container is always as wide as the longest element inside of it, this would make it dynamic and always able to hold the longest element, which might help mobile or other browsers. This could also potentially help if we decide to add longer-named routes in the future
- you could change font sizes, but then you may need to reformat how things are centered/lined-up in the navbar

**Comment by Babacar02** [comment]
@riley-thomps  I updated the branch with the recommended changes, for the time being I just increased the width by 2px. I will look into dynamically extending it, I just dont have time this week oof.

**Comment by riley-thomps** [comment]
Scrollbar looks good - will make another PR/task to make navbar dynamic

**Review by riley-thomps — APPROVED** [review]

**Pull request merged**